packages/project-management-automation/lib/tasks/pr-preview-link/index.js

```diff
@@ -0,0 +1,168 @@
+// @ts-nocheck
+
+/**
+ * Internal dependencies
+ */
+const debug = require( '../../debug' );
+
+/** @typedef {ReturnType<import('@actions/github').getOctokit>} GitHub */
+/** @typedef {any} WorkflowRunEvent */
```

Comment:
I meant to use `WebhookPayloadWorkflowRun` but is not available like `import('@octokit/webhooks').WebhookPayloadPush`, the only valid type is [WorkflowRunEvent](https://github.com/octokit/webhooks/blob/093c766b771076321bd2340816e06019c2a08de5/payload-types/schema.d.ts#L465), but I am not sure how to import it 
cc: @mirka maybe you can help with the typed issue? Thanks

Reply:
Looking at the [README](https://github.com/octokit/webhooks/tree/4abce2f5409fb3a8daa4d5c78fb6b17965c830bc/payload-types), it looks like you should import from the  `@octokit/webhooks-types` package

Have you tried doing something like

```suggestion
/** @typedef {import('@octokit/webhooks-types').WorkflowRun} WorkflowRunEvent */
```

Reply:
Thanks for suggestion @ciampo !